README.md

```diff
@@ -69,15 +78,15 @@ if($player instanceof Player && $player->isOnline()){
 	 * @phpstan-param Vector3|null $facing_pos
 	 */
 	CameraInstruction::set(
-		preset: libcamera::getPresetRegistry()->registered["target"],
+		preset: CameraPresetRegistry::TARGET(),
 		ease: new CameraSetInstructionEase(
 			CameraSetInstructionEaseType::IN_OUT_CUBIC,
 			(float) 5.0 // duration (sec)
 		),
-		camera_pos: null,
+		camera_pos: $player->getPosition()->add(0.0, $player->getEyeHeight(), 0.0), //Without it, the camera will teleport into subspace
```

Comment:
i am quite certain this is nullable, it uses the previous/current position

Reply:
I actually test everything
I think it means that once you specify it, it can be null, but in this case, since there is only one specification, it is not possible to specify null

https://github.com/user-attachments/assets/d3313973-36d6-4dd0-8184-37ae58fe1eff